resources/chsec.rb

```diff
@@ -67,23 +101,19 @@
 
 # update action
 action :update do
-  chsec_s = "chsec -f #{new_resource.name} -s #{new_resource.stanza}"
-  change = false
-  # iterating trough the hash table of sec attributes
-  new_resource.attributes.each do |key, value|
-    # checking if value has to be changed
-    if new_resource.attributes[key] == current_value.attributes[key]
-      Chef::Log.debug("chsec: value of #{key} already set to #{value} for stanza #{new_resource.stanza}")
-    else
-      change = true
-      chsec_s = chsec_s << " -a \"#{key}=#{new_resource.attributes[key]}\""
-    end
+  nr = @new_resource
```

Comment:
nr isn't a great name to use as it obscures what's happening throughout the resource.